frontend/src/views/Camps.vue

```diff
@@ -9,22 +9,23 @@
         <v-list-item
           v-for="camp in camps.items"
           :key="camp.id"
-          two-line
-          :to="campRoute(camp)">
-          <v-list-item-content>
+          two-line>
+          <v-list-item-content @click.prevent="goToCamp(camp)">
             <v-list-item-title>{{ camp.title }}</v-list-item-title>
             <v-list-item-subtitle>
               {{ camp.name }} - {{ camp.campType().organization().name }}
             </v-list-item-subtitle>
           </v-list-item-content>
           <v-list-item-action>
-            <v-btn
-              icon
-              @click.prevent="deleteCamp(camp, ...arguments)">
-              <v-icon left>
-                mdi-delete
-              </v-icon>
-            </v-btn>
+            <dialog-entity-delete :entity="camp">
+              <template v-slot:activator="{ on }">
+                <button-delete v-on="on" />
```

Comment:
```suggestion
                <button-delete @click.prevent="on.click" />
```
`v-on="on"` ist hier dasselbe wie `@click="on.click"`. Wir müssen nur noch `.prevent` hinzufügen, damit der Klick nicht auch an den Parent (das `list-item`) weitergegeben wird.